### PR TITLE
fix redirect after login failed #247

### PR DESCRIPTION
**Pull request by knoxfighter**

### CODE REVIEW

File: ui/App/views/Login.jsx [hunk 1]
```diff
@@ -22,7 +22,9 @@ const Login = ({handleLogin}) => {
                 history.push('/');
             }
         } catch (e) {
+            console.log(e);
```
Comment (mroote) [inline ui/App/views/Login.jsx:25]: Should this be removed?

File: src/bootstrap/config.go [hunk 2]
```diff
@@ -59,17 +59,22 @@ type Config struct {
 	GlibcLibLoc             string `json:"-"`
 	Autostart               string `json:"-"`
 	ConsoleCacheSize        int    `json:"console_cache_size,omitempty"` // the amount of cached lines, inside the factorio output cache
+	Secure                  bool   `json:"secure"`
```
Comment (mroote) [inline src/bootstrap/config.go:62]: Add a comment here for what the secure flag does.

**Review by mroote — COMMENTED** [review]
Looks good, left some minor comments.

File: ui/App/views/Login.jsx (hunk 1 above)
Comment (knoxfighter) [reply ui/App/views/Login.jsx:25]: No, i want to keep it in there for future problems.

Comment (mroote) [reply ui/App/views/Login.jsx:25]: sounds good

**Pull request merged**